packages/techdocs-common/src/stages/publish/awsS3.ts

```diff
@@ -131,8 +132,8 @@ export class AwsS3Publish implements PublisherBase {
       // So collecting path of only the files is good enough.
       const allFilesToUpload = await getFileTreeRecursively(directory);
 
+      const limiter = limiterFactory(10);
```

Comment:
so this is 10 per publish, not 10 per server instance - this may be the right choice, just noting

Reply:
This is a good distinction to make! I'll adjust the comments in the publishers to make it clearer 👍